### DISCUSSION TITLE
Skip tutorial

### DISCUSSION THREAD

**Discussion by Adnanali777** [SimformSolutionsPvtLtd/flutter_showcaseview#382] [General]
How can I add an option to skip the tutorial ?

**Comment by rakrisi** [comment]
did you get the solution..?

**Comment by nanox77** [comment]
Aditya-css merged it to master last week, they just need to generate the new version.

Update: version 4.0.0 is released.

**Comment by Sahil-Simform** [comment]
We have released version 4.0.0, which includes this feature. Accordingly, we are closing this discussion. You may refer to the release for further details.
